<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudyConfigurationDialog</class>
 <widget class="QDialog" name="StudyConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Study Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QTreeView" name="treeView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>15</number>
     </property>
     <widget class="QWidget" name="blankPage"/>
     <widget class="QWidget" name="amplifierPage">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTabWidget" name="amplifierHardwareTab">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Hardware</string>
          </attribute>
          <widget class="QPushButton" name="amplifierResetButton">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>90</y>
             <width>331</width>
             <height>101</height>
            </rect>
           </property>
           <property name="text">
            <string>Amplifier Hardware Reset</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ecgPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTabWidget" name="scaleComboBox1">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="displaySettingsTab">
          <attribute name="title">
           <string>Display Settings</string>
          </attribute>
          <widget class="QTableView" name="displayTableView">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>80</y>
             <width>721</width>
             <height>631</height>
            </rect>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>50</y>
             <width>721</width>
             <height>27</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Scale</string>
              </property>
              <property name="buddy">
               <cstring>scaleComboBox2</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_2">
              <item>
               <property name="text">
                <string>1/16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1/8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1/4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1/2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
           <zorder>label_3</zorder>
           <zorder>comboBox_2</zorder>
           <zorder>horizontalSpacer_2</zorder>
          </widget>
         </widget>
         <widget class="QWidget" name="hardwareSettingsTab">
          <attribute name="title">
           <string>Hardware Settings</string>
          </attribute>
          <widget class="QTableView" name="hardwareTableView">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>80</y>
             <width>721</width>
             <height>631</height>
            </rect>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>50</y>
             <width>721</width>
             <height>27</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Scale</string>
              </property>
              <property name="buddy">
               <cstring>scaleComboBox2</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="scaleComboBox2">
              <item>
               <property name="text">
                <string>1/16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1/8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1/4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1/2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pressurePage">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>210</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pressure Page</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="catheterBlockAPage"/>
     <widget class="QWidget" name="catheterBlockBPage"/>
     <widget class="QWidget" name="catheterBlockCPage"/>
     <widget class="QWidget" name="catheterBlockDPage"/>
     <widget class="QWidget" name="catheterBlockEPage"/>
     <widget class="QWidget" name="catheterBlockFPage"/>
     <widget class="QWidget" name="catheterBlockGPage"/>
     <widget class="QWidget" name="stimPage"/>
     <widget class="QWidget" name="inputChannelPage"/>
     <widget class="QWidget" name="imagingPage"/>
     <widget class="QWidget" name="analogOutPage"/>
     <widget class="QWidget" name="ablationPage">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Configuration</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Display Settings</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Real-Time</string>
                </property>
                <widget class="QGroupBox" name="groupBox_5">
                 <property name="geometry">
                  <rect>
                   <x>11</x>
                   <y>27</y>
                   <width>364</width>
                   <height>91</height>
                  </rect>
                 </property>
                 <property name="title">
                  <string>Time range (X axis)</string>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QRadioButton" name="radioButton">
                    <property name="text">
                     <string>60 s</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_2">
                    <property name="text">
                     <string>120 s</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QGroupBox" name="groupBox_6">
                 <property name="geometry">
                  <rect>
                   <x>11</x>
                   <y>132</y>
                   <width>179</width>
                   <height>181</height>
                  </rect>
                 </property>
                 <property name="title">
                  <string>Power scale</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="0" column="0">
                   <widget class="QRadioButton" name="radioButton_3">
                    <property name="text">
                     <string>75 W</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QRadioButton" name="radioButton_4">
                    <property name="text">
                     <string>100 W</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QRadioButton" name="radioButton_5">
                    <property name="text">
                     <string>125 W</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QRadioButton" name="radioButton_6">
                    <property name="text">
                     <string>150 W</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QGroupBox" name="groupBox_7">
                 <property name="geometry">
                  <rect>
                   <x>196</x>
                   <y>132</y>
                   <width>179</width>
                   <height>181</height>
                  </rect>
                 </property>
                 <property name="title">
                  <string>Number of temps</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="0" column="0">
                   <widget class="QRadioButton" name="radioButton_7">
                    <property name="text">
                     <string>1</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QRadioButton" name="radioButton_8">
                    <property name="text">
                     <string>2</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QRadioButton" name="radioButton_9">
                    <property name="text">
                     <string>3</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QRadioButton" name="radioButton_10">
                    <property name="text">
                     <string>4</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>Review (Plot Window)</string>
                </property>
                <widget class="QLabel" name="label_4">
                 <property name="geometry">
                  <rect>
                   <x>40</x>
                   <y>40</y>
                   <width>141</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Display graph for</string>
                 </property>
                </widget>
                <widget class="QWidget" name="layoutWidget">
                 <property name="geometry">
                  <rect>
                   <x>40</x>
                   <y>70</y>
                   <width>321</width>
                   <height>221</height>
                  </rect>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_9">
                  <item row="0" column="0">
                   <widget class="QCheckBox" name="checkBox">
                    <property name="text">
                     <string>Impledance</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QCheckBox" name="checkBox_5">
                    <property name="text">
                     <string>Temp1</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QCheckBox" name="checkBox_2">
                    <property name="text">
                     <string>Power</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QCheckBox" name="checkBox_6">
                    <property name="text">
                     <string>Temp 2</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QCheckBox" name="checkBox_3">
                    <property name="text">
                     <string>Voltage</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QCheckBox" name="checkBox_7">
                    <property name="text">
                     <string>Temp 3</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QCheckBox" name="checkBox_4">
                    <property name="text">
                     <string>Current</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QCheckBox" name="checkBox_8">
                    <property name="text">
                     <string>Temp 4</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <zorder>layoutWidget</zorder>
                <zorder>label_4</zorder>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Device Settings</string>
             </property>
             <widget class="QListWidget" name="listWidget">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>80</y>
                <width>256</width>
                <height>192</height>
               </rect>
              </property>
              <item>
               <property name="text">
                <string>Stockert</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>EPT</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
             </widget>
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>50</y>
                <width>151</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Ablation device type</string>
              </property>
             </widget>
             <widget class="QGroupBox" name="groupBox_8">
              <property name="geometry">
               <rect>
                <x>350</x>
                <y>50</y>
                <width>141</width>
                <height>181</height>
               </rect>
              </property>
              <property name="title">
               <string>Serial port</string>
              </property>
              <widget class="QRadioButton" name="radioButton_11">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>40</y>
                 <width>100</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>COM 1</string>
               </property>
              </widget>
              <widget class="QRadioButton" name="radioButton_12">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>80</y>
                 <width>100</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>COM 10</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QRadioButton" name="radioButton_13">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>120</y>
                 <width>100</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>None</string>
               </property>
              </widget>
             </widget>
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>560</x>
                <y>50</y>
                <width>201</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Minimum time for Ablation</string>
              </property>
             </widget>
             <widget class="QPlainTextEdit" name="plainTextEdit">
              <property name="geometry">
               <rect>
                <x>560</x>
                <y>80</y>
                <width>61</width>
                <height>31</height>
               </rect>
              </property>
              <property name="plainText">
               <string>5</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_7">
              <property name="geometry">
               <rect>
                <x>640</x>
                <y>90</y>
                <width>63</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Sec</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="measurementsPage"/>
     <widget class="QWidget" name="protocolListPage">
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>843</width>
         <height>799</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Edit Protocol List</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>70</y>
           <width>578</width>
           <height>471</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5"/>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Available Items</string>
                </property>
                <property name="buddy">
                 <cstring>availableListView</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListView" name="availableListView"/>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="selectPushButton">
                <property name="text">
                 <string>&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="selectAllPushButton">
                <property name="text">
                 <string>&gt;&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="unselectPushButton">
                <property name="text">
                 <string>&lt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="unselectAllPushButton">
                <property name="text">
                 <string>&lt;&lt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="2">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Selected Items</string>
                </property>
                <property name="buddy">
                 <cstring>selectedListView</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListView" name="selectedListView"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="activationAlignmentPage"/>
     <widget class="QWidget" name="mappingPage"/>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="saveAsButton">
     <property name="text">
      <string>Save As...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>StudyConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1109</x>
     <y>833</y>
    </hint>
    <hint type="destinationlabel">
     <x>846</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
